ra2ce/configuration/config_factory.py

```diff
@@ -66,7 +64,7 @@ def get_config_wrapper(
         return _input_config
 
     @staticmethod
-    def get_network_config_data(network_ini: Path) -> Optional[NetworkConfigData]:
+    def get_network_config_data(network_ini: Path) -> Optional[NetworkConfigWrapper]:
```

Comment:
This returns a `NetworkConfigData`, not a wrapper.

Reply:
Not sure if I agree